.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -40,9 +40,9 @@ public BaseMergedDateTimeParser(IMergedParserConfiguration configuration)
 
         public static void AddSingleDateTimeToResolution(Dictionary<string, string> resolutionDic, string type, string mod, Dictionary<string, string> res)
         {
+            // If the date is invalid, the result should be invalid.

```

Comment:
Can you make this comment more clear. Maybe include an example? I'm not sure that it actually means.
Also, what that the change in the 'if' below represent?

Reply:
"If an "invalid" Date or DateTime is extracted, it should not have an assigned resolution. Only valid entities should pass this condition."